autoinstall/README.md

```diff
@@ -18,7 +18,7 @@
 17. For UnionTech OS Server 20 1050e unattend auto-install, please use file UOS/Server/20/1050e/ks.cfg.
 18. For Fedora Server 36 or later unattend auto-install, please use file Fedora/36/Server/ks.cfg.
 19. For FreeBSD 13 or later unattend auto-install, please use file FreeBSD/installerconfig.
-20. For Pardus 21.x or later unattend auto-install, please use file Pardus/preseed.cfg.
+20. For Pardus 21.2 Server or later and 21.4 XFCE or later unattend auto-install, please use file Pardus/preseed.cfg.
```

Comment:
Is there any autoinstall support issue for Pardus 21.2 XFCE? I suggest supporting Pardus Server and XFCE both from 21.2 or 21.4. If it takes too much time to support 21.2 XFCE, I think we can claim to support server and XFCE from 21.4.  And please add Pardus into https://github.com/vmware/ansible-vsphere-gos-validation#compatible-guest-operating-systems as well.

Reply:
There is no pardus server 21.4 ISO.
Currently the newest server image is pardus 21.2.  The newest desktop image is pardus 21.4 and 21.5.
(pardus 21.5 was released 5/5/2023)